docs/girder_config_options.rst

```diff
@@ -192,6 +192,143 @@ By default, item metadata can contain any keys and values.  These can be given b
         # exclusiveMinimum: 0
         # exclusiveMaximum: 10
 
+Image View Presets
+..................
+
+This is used to specify a list of default presets for viewing images in the folder.
+Presets can be customized and saved in the GeoJS Image Viewer.
+To retrieve saved presets, use http://[serverURL]/api/v1/item/[itemID]/internal_metadata/presets.
+You can convert the response to YAML and paste it into the `imagePresets` key in your config file.
+
+Each preset can specify a name, a view mode, an image frame, and style options.
+
+- The name of a preset can be any string which uniquely identifies the preset.
+
+- There are four options for mode:
```

Comment:
When I preview this on GitHub, it renders differently than I think was intended.

Reply:
I can't find where you can look at the docs preview on Github. I've been using the VSCode preview. What looks off?

Reply:
You can just view the file directly, or navigate to the branch and the file:  (https://github.com/girder/large_image/blob/presets-from-config/docs/girder_config_options.rst)

Reply:
The section "there are four options for mode" isn't rendering in the structure I think we want.

Reply:
The sphinx-generated preview can also be seen via the circle-ci job artifacts.  Click the check or x next to the ci/circleci: list_and_docs job, choose the Artifacts tab, then build/docs.index.html.

Reply:
Even though the preview for this works well, the lint still fails it:
```
Warning, treated as error:
/root/project/docs/girder_config_options.rst:210:Unexpected indentation.
```

Reply:
You're right, I had just needed to refresh my page. Thanks!